### ISSUE TITLE
thoughts on adding qemu hypervisor 

### ISSUE THREAD

**Issue by jonnytest1** [webmin/webmin#2036]
i was thinking it would be neat to add a qemu hypervisor module and just wondering what the community thinks of this idea

**Comment by chris001** [comment]
Good idea, feel free to make a `qemu` module.  However, did you know, the [`QEMU` hypervisor is already supported in CloudMin](https://www.virtualmin.com/documentation/cloudmin/virtualization/kvm/).  You can [install CloundMin GPL from here](https://webmin.com/cloudmin/) to manage one VM for free.  Or [license CloudMin Pro](https://www.virtualmin.com/documentation/cloudmin/installation/automated/) from [here](https://www.virtualmin.com/product/cloudmin-professional-monthly-subscription/) for 10-500 virtual machines.

**Comment by shoulders** [comment]
Virtualization should be done within Cloudmin, that is it's purpose in life.

**Comment by forlotto** [comment]
Cloudmin yeah just started using it myself IDK I actually prefer the mix of webgui and CLI than the standard candy images for every operation there is something about having a terminal and a solid menu to take care of tasks that just beats other hypervisors if you ask me I've tried about 5 hypervisors I'd say cloudmin has the best interface with a slight learning curve which is alright for folks who don't do things just because they are easy.  There is always VMWare or Proxmox or Xen Orchestra for all that I suppose. But from a configuration standpoint IDK cloudmin/virtualmin ehhh dunno how much better it could be other than maybe a little upgrading and some more plugins for random stuff I guess..
